quantum-api/lib/languages/javascript.js

```diff
@@ -284,6 +284,14 @@ module.exports = (options) => {
     'function': {
       header: typeHeaderBuilders['function']
     },
+    'arg': {
+      header: typeHeaderBuilders.arg,
+      renderAsOther: { Function: typeHeaderBuilders['function'], Object: typeHeaderBuilders.object }
+    },
+    'arg?': {
+      header: typeHeaderBuilders.arg,
+      renderAsOther: { Function: typeHeaderBuilders['function'], Object: typeHeaderBuilders.object }
+    },
```

Comment:
This is the fix made that needs to be addressed in #144 

This was added so that `options` changes display in the changelog